### PR TITLE
cdc: update cdc cli doc

### PR DESCRIPTION
**Pull request by overvenus**
### What is changed, added or deleted? (Required)

Update CDC client doc.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by yikeke** [comment]
Because this week is Document Special Week, the merge button is temporarily disabled, so we can only merge this PR until the end of the Special Week (presumably this Friday night). Is it acceptable for you? @overvenus

File: ticdc/ticdc-overview.md [hunk 1]
```diff
@@ -53,6 +53,5 @@ TiCDC 的系统架构如下图所示：
 
 目前 TiCDC（4.0 发布版本）与部分 TiDB 特性存在冲突，在后续的 TiCDC 版本上会逐渐修复。当前版本需要做相应的兼容性处理。暂不支持的场景如下：
 
-- 暂不支持同步分区表。
 - 暂不支持 TiDB 4.0 [新的 Collation 框架](/character-set-and-collation.md#新框架下的-collation-支持)。如果开启该功能，需保证下游集群为 TiDB 并使用与上游相同的 collation，否则会出现 collation 导致的无法定位数据的问题。
 - 暂不支持 [TiKV Hibernate Region](https://github.com/tikv/tikv/blob/master/docs/reference/configuration/raftstore-config.md#hibernate-region)。TiCDC 会使 Region 无法进入静默状态。
```
Comment (kissmydb) [inline ticdc/ticdc-overview.md:57]: 说明一下是否能支持裸 kv？

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -93,6 +93,12 @@ aliases: ['/docs-cn/dev/reference/tools/ticdc/manage/']
     }
     ```
 
+- `admin-job-type` 代表一个 changefeed 的状态：
+    - `0`: 状态正常，也是初始状态。
+    - `1`: 任务暂停。停止任务后所有同步 `processor` 会结束退出，同步任务的配置和同步状态都会保留，可以从 `checkpoint-ts` 恢复任务。
+    - `2`: 任务恢复，同步任务从 `checkpoint-ts` 继续同步。
+    - `3`: 任务已删除，接口请求后会结束所有同步 `processor`，并清理同步任务配置信息。同步状态保留，只提供查询，没有其他实际功能。
+
 ### 管理同步子任务处理单元 (`processor`)
```
Comment (kissmydb) [inline ticdc/manage-ticdc.md:102]: CDC 的扩容缩容操作，可以添加一下？

Comment (overvenus) [reply ticdc/manage-ticdc.md:102]: 这个 PR 不涉及扩缩容，后续会有 PR 更新相关内容。

File: ticdc/ticdc-overview.md [hunk 3]
```diff
@@ -53,6 +53,5 @@ TiCDC 的系统架构如下图所示：
 
 目前 TiCDC（4.0 发布版本）与部分 TiDB 特性存在冲突，在后续的 TiCDC 版本上会逐渐修复。当前版本需要做相应的兼容性处理。暂不支持的场景如下：
 
-- 暂不支持同步分区表。
 - 暂不支持 TiDB 4.0 [新的 Collation 框架](/character-set-and-collation.md#新框架下的-collation-支持)。如果开启该功能，需保证下游集群为 TiDB 并使用与上游相同的 collation，否则会出现 collation 导致的无法定位数据的问题。
```
Comment (leoppro) [inline ticdc/ticdc-overview.md:56]:
```suggestion
- 暂不支持 TiDB 4.0 [新的 Collation 框架](/character-set-and-collation.md#新框架下的-collation-支持)。如果开启该功能，需保证下游集群为 TiDB 并使用与上游相同的 collation，否则会出现 collation 导致的无法定位数据的问题。
- 暂不支持 TiDB 4.0 [SEQUENCE DDL](/sql-statements/sql-statement-create-sequence/) 和 [SEQUENCE 函数](/sql-statements/sql-statement-create-sequence/#sequence-函数)。在上游 TiDB 使用 SEQUENCE 时，TiCDC 将会忽略掉上游执行的 SEQUENCE DDL/函数，但是使用 SEQUENCE 函数的 DML 可以正确的同步。
```

Comment (overvenus) [reply ticdc/ticdc-overview.md:56]: Thanks!

File: ticdc/ticdc-overview.md [hunk 4]
```diff
@@ -53,6 +53,6 @@ TiCDC 的系统架构如下图所示：
...
+- 暂不支持 TiDB 4.0 [SEQUENCE DDL](/sql-statements/sql-statement-create-sequence/) 和 [SEQUENCE 函数](/sql-statements/sql-statement-create-sequence/#sequence-函数)。在上游 TiDB 使用 SEQUENCE 时，TiCDC 将会忽略掉上游执行的 SEQUENCE DDL/函数，但是使用 SEQUENCE 函数的 DML 可以正确的同步。
```
Comment (TomShawn) [inline ticdc/ticdc-overview.md:57]:
```suggestion
- 暂不支持 TiDB 4.0 [SEQUENCE DDL](/sql-statements/sql-statement-create-sequence.md) 和 [SEQUENCE 函数](/sql-statements/sql-statement-create-sequence.md#sequence-函数)。在上游 TiDB 使用 SEQUENCE 时，TiCDC 将会忽略掉上游执行的 SEQUENCE DDL/函数，但是使用 SEQUENCE 函数的 DML 可以正确的同步。
```

Comment (overvenus) [reply ticdc/ticdc-overview.md]: Thank you!

File: ticdc/manage-ticdc.md [hunk 5]
```diff
@@ -200,20 +206,12 @@ election: not leader
...
 ```shell
-curl -X POST -d "admin-job=1&cf-id=28c43ffc-2316-4f4f-a70b-d1a7c59ba79f" http://127.0.0.1:8301/capture/owner/admin
-```
-
-```
-{
- "status": true,
- "message": ""
-}
+cdc cli changefeed --changefeed-id 28c43ffc-2316-4f4f-a70b-d1a7c59ba79f pause
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:209]: 这个不应该属于这一章：`使用 HTTP 接口管理集群状态和数据同步`

File: ticdc/manage-ticdc.md [hunk 6]
```diff
@@ -37,61 +37,115 @@ aliases: ['/docs-cn/dev/reference/tools/ticdc/manage/']
...
+                "resolved-ts": 415241860902289409,
+                "checkpoint-ts": 415241860640145409,
+                "admin-job-type": 0
+        }
+}
+```
+
+以上命令中：
+
+- `admin-job-type` 代表一个 changefeed 的状态：
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:104]: 建议再解释一下resolved-ts和checkpoint-ts，曾经有用户问过我

File: ticdc/ticdc-overview.md [hunk 7]
```diff
@@ -53,6 +53,7 @@ TiCDC 的系统架构如下图所示：
 
 目前 TiCDC（4.0 发布版本）与部分 TiDB 特性存在冲突，在后续的 TiCDC 版本上会逐渐修复。当前版本需要做相应的兼容性处理。暂不支持的场景如下：
 
-- 暂不支持同步分区表。
+- 暂不支持同步使用 RawKV 的 TiKV 集群
```
Comment (leoppro) [inline ticdc/ticdc-overview.md:56]: 目前我们release的版本似乎仍然不支持分区表

**Comment by yikeke** [comment]
PTAL again @TomShawn @kissmydb

File: ticdc/manage-ticdc.md [hunk 8]
```diff
@@ -37,61 +37,115 @@ aliases: ['/docs-cn/dev/reference/tools/ticdc/manage/']
...
+```
+[
+        {
+                "id": "28c43ffc-2316-4f4f-a70b-d1a7c59ba79f"
+        }
+]
+```
 
-    {{< copyable "shell-regular" >}}
+#### 查询特定同步任务，对应于某个同步任务的信息和状态
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:69]:
```suggestion
#### 查询特定同步任务
```

File: ticdc/manage-ticdc.md [hunk 9]
```diff
@@ -37,61 +37,115 @@ aliases: ['/docs-cn/dev/reference/tools/ticdc/manage/']
...
 
-    ```shell
-    cdc cli changefeed query --changefeed-id=28c43ffc-2316-4f4f-a70b-d1a7c59ba79f
-    ```
+使用以下命令来查询特定同步任务：
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:71]:
```suggestion
使用以下命令来查询特定同步任务（对应某个同步任务的信息和状态）：
```

File: ticdc/manage-ticdc.md (hunk 2 above)
Comment (kissmydb) [reply ticdc/manage-ticdc.md:150]: OK

File: ticdc/ticdc-overview.md [hunk 10]
```diff
@@ -54,5 +54,7 @@ TiCDC 的系统架构如下图所示：
 目前 TiCDC（4.0 发布版本）与部分 TiDB 特性存在冲突，在后续的 TiCDC 版本上会逐渐修复。当前版本需要做相应的兼容性处理。暂不支持的场景如下：
 
 - 暂不支持同步分区表。
+- 暂不支持同步使用 RawKV 的 TiKV 集群。
```
Comment (kissmydb) [inline ticdc/ticdc-overview.md:57]:
```suggestion
- 暂不支持单独使用 RawKV 的 TiKV 集群。
```

File: ticdc/manage-ticdc.md [hunk 11]
```diff
@@ -37,61 +37,115 @@ aliases: ['/docs-cn/dev/reference/tools/ticdc/manage/']
...
+                "resolved-ts": 415241860902289409,
+                "checkpoint-ts": 415241860640145409,
+                "admin-job-type": 0
+        }
+}
+```
+
+以上命令中：
+
+- `admin-job-type` 代表一个 changefeed 的状态：
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:104]:
```suggestion
- `resolved-ts` 代表当前 changfeed 中最大的已经成功从 TiKV 发送到 TiCDC 的事务 TS；
- `checkpoint-ts` 代表当前 changefeed 中最大的已经成功写入下游的事务 TS；
- `admin-job-type` 代表一个 changefeed 的状态：
```

**Pull request merged**